File: RetroGameGauntlet.iOS/Renders/ColoredTabbedPageRenderer.cs
using Xamarin.Forms.Platform.iOS;
using Xamarin.Forms;
using RetroGameGauntlet.iOS.Renders;

[assembly: ExportRenderer(typeof(TabbedPage), typeof(ColoredTabbedPageRenderer))]
namespace RetroGameGauntlet.iOS.Renders
{
    public class ColoredTabbedPageRenderer : TabbedRenderer 
    {
        public ColoredTabbedPageRenderer ()
        {
            TabBar.TintColor = Color.White.ToUIColor();
            TabBar.BarTintColor = ((Color)Xamarin.Forms.Application.Current.Resources["backgroundColor"]).ToUIColor();
            TabBar.BackgroundColor = Color.Blue.ToUIColor();
        }
    }
}

File: RetroGameGauntlet.iOS/Renderers/ColoredTabbedPageRenderer.fs
namespace RetroGameGauntlet.iOS.Renderers

open System
open UIKit
open Xamarin.Forms
open Xamarin.Forms.Platform.iOS

type ColoredTabbedPageRenderer () as this =
    inherit TabbedRenderer ()

    do
        let backgroundColor = Application.Current.Resources.Item "backgroundColor" :?> Color
        let textColor = Application.Current.Resources.Item "textColor" :?> Color

        this.TabBar.TintColor <- ColorExtensions.ToUIColor textColor
        this.TabBar.BarTintColor <- ColorExtensions.ToUIColor backgroundColor
        this.TabBar.BackgroundColor <- UIColor.Blue

        let fontName = Application.Current.Resources.Item "monoFont" :?> string
        let fontSize = nfloat 8.0
        let font = UIFont.FromName(fontName, fontSize)
        let textAttributes = new UITextAttributes(Font=font)
        UITabBarItem.Appearance.SetTitleTextAttributes(textAttributes, UIControlState.Normal)

[<assembly: ExportRenderer(typeof<TabbedPage>, typeof<ColoredTabbedPageRenderer>)>] do ()